docs/content/manual/_incoming/3045-upgrade-with-bonded-nic.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade Harvester on node that has bonded NICs for management interface
+---
+
+* Related issues: [#3045](https://github.com/harvester/harvester/issues/3045) [BUG] Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface
+
+## Category: 
+* Upgrade
+
+## Environment Setup
+
+- This is to be done on a Harvester cluster where the NICs were configured to be bonded on install for the management interface
+
+## Verification Steps
+
+1. Install Harvester with at least 2 bonded NICs on install in `balance-tlb`
+
+    ![image](https://user-images.githubusercontent.com/83787952/198138366-472d4432-839a-4d9d-a942-d55fe8f6f6d8.png)
```

Comment:
I think this picture is great! :smile:   I really dig it's use here to be able to help show what the installation screen should hold.
It renders really well, when I tested locally with Hugo - 
![Screenshot from 2022-11-02 10-41-04](https://user-images.githubusercontent.com/5370752/199563331-ee518770-6c5b-4d6a-97eb-ecaa5fd614e9.png)